tools/container/toolkit/toolkit.go

```diff
@@ -404,7 +417,65 @@ func Install(cli *cli.Context, opts *options) error {
 		log.Errorf("Ignoring error: %v", fmt.Errorf("error installing NVIDIA Container CDI Hook CLI: %v", err))
 	}
 
-	err = installToolkitConfig(cli, toolkitConfigPath, nvidiaContainerCliExecutable, nvidiaCTKPath, nvidiaContainerRuntimeHookPath, opts)
+	var runtimeBinaryPaths []string
+
+	if r, ok := os.LookupEnv("RUNTIME"); ok {
+
+		switch r {
+		case "containerd":
+			var cliArgs []string
+			if opts.HostRootMount != "" {
+				cliArgs = append(cliArgs, "chroot", opts.HostRootMount)
+			}
+			cliArgs = append(cliArgs, "containerd", "config", "dump")
+
+			cfg, err := containerd.New(
+				containerd.WithReference(pkgconfig.FromCLI(cliArgs)),
+			)
+			if err != nil {
+				return fmt.Errorf("unable to load containerd config: %w", err)
+			}
+
+			containerdDefaultRuntime := cfg.DefaultRuntime()
```

Comment:
We don't just want this for the default runtime, or are we sure that `DefaultRuntime` will always be set.